skan/test/test_csr.py

```diff
@@ -4,6 +4,7 @@
 from skan import csr
 
 rundir = os.path.dirname(os.path.abspath(__file__))
+print (rundir)
```

Comment:
```suggestion
print(rundir)
```
in python3 `print` is a function :)